tests/ossfuzz/config/fuzz_lzma.dict

```diff
@@ -0,0 +1,20 @@
+# first 5 header bytes of .lzma archives based on the info from
+# - extra/scanlzma/scanlzma.c (lines 32-52)
+# - src/liblzma/common/alone_encoder.c (lines 107-126)
+# - https://github.com/cscott/lzma-purejs/blob/master/FORMAT.md
+# - manual hexdump of data produced by the lzma CLI suite
+
+# compression mode 1    (LZMA properties = 0x5d, dict size = 0x00100000)
```

Comment:
Instead of **mode** it should be **preset**. In the documentation **mode** in the context of compression mode refers to a match finder setting (fast or normal).

The LZMA properties resulting in 0x5d are lc=3, lp=0, pb=2. It could be helpful to mention this once, and then only mention the dictionary size for each .dict entry since the LZMA properties aren't changing.